bench/hamming-cleanup/quadratic.fpcore

```diff
@@ -0,0 +1,127 @@
+; -*- mode: scheme -*-
+
+(FPCore (a b c)
+ :name "quadp (p42, positive)"
+ ;:herbie-expected #f ; Fails in single precision due to bad samples
+ :herbie-target
+ (let ([sqtD
+  (let ([x (* (sqrt (fabs a)) (sqrt (fabs c)))])
+    (if (or (and (> a 0) (> c 0)) (and (<= a 0) (<= c 0)))
```

Comment:
I think this is fine but there's a slicker way of writing this: `(if (== (copysign a b) a)`